modules/fbs/includes/fbs.availability.inc

```diff
@@ -66,7 +66,7 @@ function fbs_availability_holdings($provider_ids) {
       'is_periodical' => FALSE,
     );
 
-    $placement_parts = array('branch', 'department', 'location', 'sublocation');
+    $placement_parts = array('branch', 'department', 'location', 'sublocation', 'materials');
```

Comment:
If the line declaring an array spans longer than 80 characters, each element should be broken into its own line.